AsyncDisplayKit/ASTableView.h

```diff
@@ -141,6 +141,13 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)relayoutItems;
 
 /**
+ * Triggers a recalulation of leadingScreensForBatching
+ * @discussion During various layout changes that trigger content size changes, batch fetching needs to recalulate it's number of screens left to scroll before the delegate -collectionView:beginBatchFetchingWithContext: is called. Call when a layout transition is complete, or it will have no effect.
+ *
+ */
+- (void)invalidateBatchFetchMeasurements;
```

Comment:
I think it's too unclear to users when and why they need to call this, at least because it is still unclear to me :).  It may indeed be the only solution, but we have to clarify the cases that it is needed; e.g. what's an example using ASTableView?

I think we should find any possible way for this to be triggered internally even if it means that batch fetching will be re-checked more frequently than absolutely necessary.